app/uploaders/validate_pdf.rb

```diff
@@ -22,6 +22,9 @@ def validate(temp_file)
   end
 
   def validate_pdf(file)
+    # Skip when the pdf file is generated by DisabilityCompensationFastTrackJob
```

Comment:
@julialeague Are we skipping because DisabilityCompensationFastTrackJob doesn't need to create a `temp_file`?